components/prism-dot.js

```diff
@@ -0,0 +1,77 @@
+// https://www.graphviz.org/doc/info/lang.html
+
+(function (Prism) {
+
+	var ID = '(?:' + [
+		// an identifier
+		/\b[a-zA-Z_][\w\x80-\x9F\xA1-\xFF]*/.source,
+		// a number
+		/-?(?:\.\d+|\b\d+(?:\.\d*)?)/.source,
+		// a double-quoted string
+		/"(?:[^\\"]|\\[\s\S])*"/.source,
+		// HTML-like string
+		/<(?:[^<>]|(?!<!--)<(?:[^<>"']|"[^"]*"|'[^']*')+>|<!--(?:[^-]|-(?!->))*-->)*>/.source
+	].join('|') + ')';
+
+	var IDInside = {
+		'markup': {
+			pattern: /(^<)[\s\S]+(?=>$)/,
+			lookbehind: true,
+			alias: ['language-markup', 'language-html', 'language-xml'],
+			inside: Prism.languages.markup
+		}
+	};
+
+	/**
+	 * @param {string} source
+	 * @param {string} flags
+	 * @returns {RegExp}
+	 */
+	function withID(source, flags) {
+		return RegExp(source.replace(/<ID>/g, function () { return ID; }), flags);
+	}
+
+	Prism.languages.dot = {
+		'comment': {
+			pattern: /\/\/.*|\/\*[\s\S]*?\*\/|^#.*/m,
+			greedy: true
+		},
+		'graph-name': {
+			pattern: withID(/(\b(?:digraph|graph|subgraph)\s+)<ID>/.source, 'i'),
+			lookbehind: true,
+			greedy: true,
+			alias: 'class-name',
+			inside: IDInside
+		},
+		'attr-value': {
+			pattern: withID(/(=\s*)<ID>/.source),
+			lookbehind: true,
+			greedy: true,
+			inside: IDInside
+		},
+		'attr-name': {
+			pattern: withID(/([\[;,\s])<ID>(?=\s*=)/.source),
+			lookbehind: true,
+			greedy: true,
+			inside: IDInside
+		},
+		'keyword': /\b(?:digraph|edge|graph|node|strict|subgraph)\b/i,
+		'compass-point': {
+			pattern: /(:\s*)(?:[ns][ew]?|[ewc])(?![\w\x80-\x9F\xA1-\xFF])/,
```

Comment:
> The compass point "_" specifies that an appropriate side of the port adjacent to the exterior of the node should be used, if such exists.

(from <https://graphviz.org/doc/info/attrs.html>).

> compass_pt : (n | ne | e | se | s | sw | w | nw | c | _)

(from <https://graphviz.org/doc/info/lang.html>)

An example:

```dot
digraph {
  a -> b:_;
}
```

Reply:
I forgot one... Thanks for pointing that out!

Reply:
Glad to help!